internal/operators/mce/mce_operator.go

```diff
@@ -198,12 +198,14 @@ func (o *operator) GetPreflightRequirements(context context.Context, cluster *co
 	}, nil
 }
 
+/*
```

Comment:
How come this is commented out? :o

Reply:
This is work in progress. I need to comment it out to be able to test without an smaller number of nodes. I will make the pull request a draft.

Reply:
Done.